xmtp_mls/src/groups/mls_sync.rs

```diff
@@ -1556,13 +1560,39 @@ where
 
                         Ok(updates)
                     })?;
+            let extensions: Extensions = mls_group.extensions().clone();
+            let old_group_membership = extract_group_membership(&extensions)?;
+            let mut new_membership = old_group_membership.clone();
+            for (inbox_id, sequence_id) in changed_inbox_ids.iter() {
+                new_membership.add(inbox_id.clone(), *sequence_id);
+            }
+
+            let changes_with_kps = calculate_membership_changes_with_keypackages(
+                &self.client,
+                provider,
+                &new_membership,
+                &old_group_membership,
+            )
+            .await?;
+
+            // If we fail to fetch or verify all the added members' KeyPackage, return an error.
+            // skip if the inbox ids is 0 from the beginning
+            //todo: get the installations for all!
+            if !inbox_ids_to_add.is_empty()
+                && changes_with_kps.failed_installations.len() == inbox_ids_to_add.len()
+            {
+                return Err(GroupError::Generic(
+                    "Failed to add all installations".to_string(),
```

Comment:
Would it be more accurate to say "Failed to verify all installations" instead? Just thinking about reading this error in future logs, and what would make it easier to comprehend.